community/github-config.yaml

```diff
@@ -1138,6 +1138,110 @@ orgs:
           user-api: triage
           website-tooling: triage
           workflow-helpers: triage
+      SigChairs:
```

Comment:
isnt 'write' permission less than the 'triage' that all the devs team members have?

Reply:
no, `write` allows more than `triage`: https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role

Reply:
you are right, I was misinterpreting the table layout